app.json

```
@@ -20,6 +20,10 @@
       "description": "Comma separated list of labels to include",
       "required": false
     },
+    "GITHUB_SECURITY_ALERTS": {
+      "description": "Should Dependapanda include security alerts",
```

Based on your commit message I'd tweak this to:

```suggestion
      "description": "Should Dependapanda include security alerts information along with the standard Dependapanda info",
```